apps/xmtp.chat/src/components/Messages/Message.tsx

```diff
@@ -23,7 +28,10 @@ export const Message: React.FC<MessageProps> = ({
   const isSender = client.inboxId === message.senderInboxId;
   const align = isSender ? "right" : "left";
   const isActionable =
```

Comment:
How about we also put an `isActionable(message)` in the `helpers/messages.ts` file?

Reply:
sounds good